include/deal.II/fe/fe.h

```diff
@@ -2275,6 +2275,16 @@ class FiniteElement : public Subscriptor, public FiniteElementData<dim>
 
   //@}
 
+  /**
+   * For a finite element of degree @p sub_degree < @p degree, we return a
+   * vector which maps the numbering on an FE of degree @p sub_degree into the
+   * numbering on this element. This function is appropriate for hierarchical
```

Comment:
```suggestion
   * numbering on this element.
   *
   * This function is appropriate for hierarchical
```
Maybe separate this into two paragraphs.